src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexRunnerFactory.cs

```diff
@@ -83,14 +84,13 @@ private sealed class Runner<TSetType> : RegexRunner where TSetType : notnull
 
             internal Runner(SymbolicRegexMatcher<TSetType> matcher) => _matcher = matcher;
 
-            protected override void InitTrackCount() { } // nop, no backtracking
-
-            protected override bool FindFirstChar() => true; // The logic is all in Go.
-
-            protected override void Go()
+            protected internal override void Scan(ReadOnlySpan<char> text)
             {
-                ReadOnlySpan<char> inputSpan = runtext;
+                Go(text);
+            }
 
+            private void Go(ReadOnlySpan<char> inputSpan)
+            {
```

Comment:
We don't need Go to be a separate method.  The body of Go can just be the body of Scan.

Reply:
totally true, originally I had it separate as in Scan I had a lot of the boilerplate that I had for other engine's Scan methods, but I then started removing all of that since it didn't apply for this engine and was left with just the invocation of Go. Will fix this.